ui/views.py

```diff
@@ -172,7 +172,9 @@ def get_vocabularies(facets):
         vocab = (vocabulary_id, vocabs[int(vocabulary_id)])
         vocabularies[vocab] = []
         for t_id, count in term_data:
```

Comment:
On deleting a term, that particular term get delete from **Term model**. But not from **term_data**. Can any one help me understanding this. I just added a check (L176)
 > ``` if t_id_int in terms: ```